doc/whats_new.rst

```diff
@@ -53,6 +53,8 @@ BUG
 
     - Fix channel name pick in :func:`mne.Evoked.get_peak` method by `Alex Gramfort`_
 
+    - Fix drop percentages to take into account ``ignore`` option in :func:`mne.viz.plot_drop_log` and :func:`mne.Epochs.plot_drop_log` by `Eric Larson`_.
```

Comment:
The former is a function and can operate on list of lists, second is a method of the `Epochs` class, not a function in the `mne.epochs` namespace. Make sense?